fault-monitor/operational-status-monitor.hpp

```diff
@@ -91,6 +91,20 @@ class Monitor
      */
     void updateAssertedProperty(const std::vector<std::string>& ledGroupPaths,
                                 bool value);
+
+    /**
+     * @brief API to remove chassis critical association on the d-bus object
+     *
+     * This method removes the critical association between the given object
+     * path and chassis when the corresponding faulty FRU is repaired/replaced.
+     * The repair status of the FRU can be identified from the FRU's operational
+     * status (Functional property). The critical association is removed when
```

Comment:
just a single line that "chassis association is removed when operational status is set to true" should be good enough. extra data causes issues.